Source/santametricservice/Writers/SNTMetricFileWriter.m

```diff
@@ -0,0 +1,77 @@
+/// Copyright 2021 Google Inc. All rights reserved.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+///    Unless required by applicable law or agreed to in writing, software
+///    distributed under the License is distributed on an "AS IS" BASIS,
+///    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///    See the License for the specific language governing permissions and
+///    limitations under the License.
+
+#import "Source/santametricservice/Writers/SNTMetricFileWriter.h"
+#import "Source/common/SNTLogging.h"
+
+@implementation SNTMetricFileWriter
+
+/*
+ * Open a file for appending.
+ */
+- (NSFileHandle *)fileHandleForAppendingAtPath:(NSString *)path createMode:(mode_t)mode {
```

Comment:
I'd argue the name of this is incorrect as you're opening with O_TRUNC. How about `fileHandleForNewFileAtPath:createMode:`?

Reply:
Agreed, it's renamed.